rest/javascript/evm_client/main.js

```diff
@@ -84,7 +84,7 @@ async function callContract(response) {
     };
 
     // Sign the transaction with the private key
-    const signedTransaction = await web3.eth.accounts.signTransaction(transactionObject, "<PRIVATE KEY>");
+    const signedTransaction = await web3.eth.sendSignedTransaction(signedTransaction.rawTransaction,null,{checkRevertBeforeSending:false});
```

Comment:
This will not work
you need to add this on line 90.
keep this signing as it is.